packages/lesswrong/components/messaging/MessagesNewForm.tsx

```diff
@@ -27,15 +27,16 @@ const styles = (theme: ThemeType) => ({
   },
   rootMinimalist: {
     ...theme.typography.commentStyle,
-    padding: 10,
+    padding: 0,
```

Comment:
1. The loss of padding here makes it look not great when you hover over the button
<img width="1484" height="984" alt="Screenshot 2025-10-06 at 20 42 02" src="https://github.com/user-attachments/assets/926ad668-2b93-4641-93a4-9ece41d05816" />
2. I may be missing something but it looks like the colour has also changed? This is staging currently:
<img width="1484" height="984" alt="Screenshot 2025-10-06 at 20 44 07" src="https://github.com/user-attachments/assets/58de4a49-e661-4e51-a387-af7baddcd285" />